tests/TestCase.php

```diff
@@ -67,10 +67,13 @@ public function call($method, $uri, $parameters = [], $cookies = [], $files = []
         return parent::call($method, $uri, $parameters, $cookies, $files, $server, $content);
     }
 
-    public function json($method, $uri, array $data = [], array $headers = [], $options = 0, ?VersionEnum $apiVersion = null): TestResponse
+    public function json($method, $uri, array $data = [], array $headers = [], $options = 0, VersionEnum|bool|null $apiVersion = null): TestResponse
```

Comment:
please use `get` method for the status tests instead of extending `json` method

Reply:
Done.